docs/core/compatibility/8.0.md

```diff
@@ -27,9 +27,10 @@ If you're migrating an app to .NET 8, the breaking changes listed here might aff
 
 | Title                                                                   | Type of change    |
 | ----------------------------------------------------------------------- | ----------------- |
-| ['ca-certificates' and 'krb5-libs' packages removed from Alpine images](containers/8.0/krb5-libs-package.md) | Binary incompatible |
+| ['ca-certificates' package removed from Alpine images](containers/8.0/ca-certificates-package.md) | Binary incompatible |
 | [Debian container images upgraded to Debian 12](containers/8.0/debian-version.md) | Binary incompatible/behavioral change |
 | [Default ASP.NET Core port changed to 8080](containers/8.0/aspnet-port.md) | Behavioral change |
+| [Kerberos package removed from Alpine and Debian images](containers/8.0/krb5-libs-package.md) | Binary incompatible |
```

Comment:
Should this match the title from the document below?

`Kerberos package removed from .NET container images`

Reply:
I think I'm missing something obvious. Elaborate?

Reply:
The title metadata and the H1 are different in that article:

<img width="324" alt="image" src="https://github.com/dotnet/docs/assets/24882762/f2caf503-bc09-4e2e-aa6a-4d64a6b0f2ca">

Reply:
OK. I think I fixed it now. Thanks for the explanation.